code/style.css

```diff
@@ -1,7 +1,267 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+}
+
+main, header, section, aside, footer {
+  margin: 0;
+  border: none;
+  color: #5B5B5B ;
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: small;
+}
+
+main {
+  background: #FFFFFF;
+}
+
+.header {
+  background: #AAD6EC;
+  padding: 20px;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: center;
+}
+
+.hero {
+  background: #5B5B5B ;
+  color: #FFFFFF ;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  /* position: relative; */
+}
+
+.hero-image {
+  width: 100vw;
```

Comment:
I understand the initial idea that hero img has to fill the max view port possible . Unfortunately it did not work as expected . When you inspect the page and/or open netlify link on mobile and tablet , it is visible that the pics are not responsive and it creates a large blank areas and horizontal scroll on these layouts.